GenericsAPI.spec

```diff
@@ -129,8 +130,8 @@ module GenericsAPI {
       workspace_name workspace_name;
 
       obj_ref genome_ref;
-      obj_ref col_conditionset_ref;
-      obj_ref row_conditionset_ref;
+      obj_ref col_attributemapping_ref;
```

Comment:
we probably also want to update UI to reflect the changes.

Reply:
Updated the UI.